src/components/environments-summary/environment-card.tsx

```diff
@@ -191,13 +196,24 @@ export const EnvironmentCard = ({
         )}
 
         {body}
-        {activeDeployment(appName, env)}
+        {activeDeployment(appName, deployment)}
 
-        <Typography group="ui" variant="chip__badge">
-          {!env.branchMapping
-            ? 'Not built automatically'
-            : `Built from ${env.branchMapping} branch`}
-        </Typography>
+        <div className="grid">
+          <Typography group="ui" variant="chip__badge">
+            {env.branchMapping
+              ? `Built from ${env.branchMapping} branch`
+              : 'Not built automatically'}
+          </Typography>
+          {deployment?.gitTags && (
+            <div className="grid grid--gap-x-small grid--auto-columns">
+              <Icon data={github} size={18} />
+              <GitTagLinks
```

Comment:
The font size for tags seems a bit large compared to the rest of the text in the environment card

Reply:
adjusted